CHANGELOG.md

```diff
@@ -756,8 +760,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Hide unpublished pages from public version of organizations list page
 - Improve sanitizing of input text on CKEditor plugin
-- Replace course-detail__run-cta in fragment_course_run template by
-  course-run-enrollment__cta
+- Replace course-detail**run-cta in fragment_course_run template by
+  course-run-enrollment**cta
```

Comment:
oups. You have to hold your linter tighter! Maybe remove all reformatting from your commit and eventually add a separated commit for the legitimate ones?